server/src/utils/MixerConnection.ts

```diff
@@ -416,161 +412,65 @@ export class MixerGenericConnection {
         fadeTime: number,
         faderIndex: number
     ) => {
-        let outputLevel =
+        const outputLevel =
             state.channels[0].chMixerConnection[mixerIndex].channel[
                 channelIndex
-            ].outputLevel
+                ].outputLevel
         let targetVal = state.faders[0].fader[faderIndex].faderLevel
 
         if (state.faders[0].fader[faderIndex].voOn) {
             targetVal = (targetVal * (100 - state.settings[0].voLevel)) / 100
         }
-        const step: number =
-            (targetVal - outputLevel) / (fadeTime / FADE_INOUT_SPEED)
-        const dispatchResolution: number =
-            this.mixerProtocol[mixerIndex].FADE_DISPATCH_RESOLUTION * step
-        let dispatchTrigger: number = 0
+
+        this.fade(fadeTime, mixerIndex, channelIndex, outputLevel, targetVal)
+    }
+
+    fade = (fadeTime: number, mixerIndex: number, channelIndex: number, startLevel: number, endLevel: number) => {

```

Comment:
```suggestion
    fade(fadeTime: number, mixerIndex: number, channelIndex: number, startLevel: number, endLevel: number) {
```
Just a suggestion to use a function declaration instead of a lambda.